projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -678,6 +840,49 @@ export class OwcJsonService {
     return displayName;
   }
 
+  getResourceDescription(resource: IOwsResource): string {
+    let description = '';
+    if (resource.properties.abstract) {
+      description = resource.properties.abstract;
+    }
+    return description;
+  }
+
+  getLayerElevationDimension(elevationDimension: IEocOwsResourceDimension): ILayerElevationDimension {
+    throw new Error('Method not implemented.');
+  }
+
+  getLayerTimeDimension(time: IEocOwsResourceDimension): ILayerTimeDimension {
+    if (time.name !== 'time') {
+      console.error('Not a time-dimension: ', time);
+      return;
+    }
+
+    const out: ILayerTimeDimension = {
+      units: time.units,
+      values: null
+    };
+
+    if (time.values.includes(',')) {
+      const values = time.values.split(',');
+      out.values = values;
+
+    } else if (time.values.includes('/')) { // period
+      const matches = time.values.match(/\d\d\d\d-\d\d-\d\d(T\d\d:\d\d:\d\d.\d\d\dZ)*/gm);
```

Comment:
Parsing ISO Time is not easy https://stackoverflow.com/a/14322189/10850021 for this we should maybe us a library like `luxon` ur let it to the application developer.
In the long run hopefully we can use https://tc39.es/proposal-temporal/docs/parse-draft.html

Reply:
Right; Will adjust code to use luxon.

Reply:
Code has been adjusted to make use of luxon for parsing ISO8601 strings.
Note, however, that the type signature of `getTimeValueFromDimensions` requires us to return (an object containing) ISO8601-strings again. Effectively, therefore, we're parsing ISO with luxon and then have luxon output ISO again.
The only gain from luxon is that we no longer use our own, custom Regexes.

Reply:
... except in parseISO8601Period, because luxon does [not know about periodically repeating measurements within a duration](https://github.com/moment/luxon/search?q=period), it seems (at least I couldn't find such a thing)

Reply:
Then maybe the help of `luxon` is only to check if the provided strings are valid.
I just wanted to avoid creating our own regular expressions to parse / validate ISO.